build-aux/com.usebottles.bottles.json

```diff
@@ -402,4 +317,4 @@
             ]
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
This is unrelated

Reply:
its automatic formatting, idk how to not do, not a big deal but i can look into if needed

Reply:
I guess it depends on what @mirkobrombin thinks. Personally, to avoid polluting `git blame`, I prefer not to have unrelated changes

Reply:
We can keep it for this time